x/feegrant/client/cli/tx.go

```diff
@@ -55,9 +55,9 @@ func NewCmdFeeGrant() *cobra.Command {
 				ignored as it is implied from [granter].
 
 Examples:
-%s tx %s grant cosmos1skjw... cosmos1skjw... --spend-limit 100stake --expiration 36000 or
+%s tx %s grant cosmos1skjw... cosmos1skjw... --spend-limit 100stake --expiration 2022-01-02T15:04:05Z or
 %s tx %s grant cosmos1skjw... cosmos1skjw... --spend-limit 100stake --period 3600 --period-limit 10stake --expiration 36000 or
-%s tx %s grant cosmos1skjw... cosmos1skjw... --spend-limit 100stake --expiration 36000 
+%s tx %s grant cosmos1skjw... cosmos1skjw... --spend-limit 100stake --expiration 2022-01-02T15:04:05Z 
```

Comment:
Should we mention format as well? Some would be confused b/w YYYY-MM-DD with YYYY-DD-MM. or else let's change the example date so we can differntiate b/w month, date easily

Reply:
the format is mentioned as part of the flag `FlagExpiration` description but good point, I'll update the example with less confusing dates